.pre-commit-config.yaml

```diff
@@ -20,56 +22,44 @@ repos:
       - id: check-merge-conflict
       - id: check-toml
       - id: check-vcs-permalinks
-      - id: check-yaml
       - id: detect-private-key
         exclude: ^pkg/k8s/secrets/certs\.go$|^pkg/syncthing/certs\.go$|^cmd/deploy/certs\.go$
       - id: end-of-file-fixer
       - id: fix-byte-order-marker
       - id: mixed-line-ending
       - id: trailing-whitespace
         exclude: ^\.github/pull_request_template\.md$
+
+  # shell and markdown checks
   - repo: https://github.com/jumanjihouse/pre-commit-hooks
     rev: 2.1.6
     hooks:
       - id: shellcheck
-      # - id: shfmt
+      - id: markdownlint
+
+  # check spelling though all the repository
   - repo: https://github.com/codespell-project/codespell
     rev: v2.1.0
     hooks:
       - id: codespell
         name: Run codespell
         description: Check Spelling with codespell
-        entry: codespell --exclude-file=".circleci/config.yml" --ignore-words=codespell.txt
+        entry: codespell -L flate,keypair,splitted,vertexes,ba
+
   - repo: https://github.com/pre-commit/mirrors-prettier
     rev: v2.6.1
     hooks:
       - id: prettier
         name: Run prettier
         description: Format files with prettier
-  - repo: https://github.com/igorshubovych/markdownlint-cli
-    rev: v0.31.1
-    hooks:
-      - id: markdownlint
-        name: Run markdownlint
-        description: Check Markdown files with markdownlint
-        entry: markdownlint -c .github/linters/.markdown-lint.yml .
-        types: [markdown]
-        files: \.(md|mdown|markdown)$
+
+  # yaml files check
   - repo: https://github.com/adrienverge/yamllint
     rev: v1.26.3
     hooks:
       - id: yamllint
```

Comment:
Isn't this covered by markdownlint in line 38: https://github.com/okteto/okteto/pull/3036/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R38